lib/src/main/java/uk/gov/di/ipv/core/library/service/AuditService.java

```diff
@@ -24,21 +25,24 @@ public static AmazonSQS getDefaultSqsClient() {
         return AmazonSQSClientBuilder.defaultClient();
     }
 
-    public void sendAuditEvent(AuditEventTypes eventType) throws SqsException {
+    public void sendAuditEvent(AuditEventTypes eventType, AuditExtensionParams extensions)
+            throws SqsException {
         try {
             SendMessageRequest sendMessageRequest =
                     new SendMessageRequest()
                             .withQueueUrl(queueUrl)
-                            .withMessageBody(generateMessageBody(eventType));
+                            .withMessageBody(generateMessageBody(eventType, extensions));
 
             sqs.sendMessage(sendMessageRequest);
         } catch (JsonProcessingException e) {
             throw new SqsException(e);
         }
     }
 
-    private String generateMessageBody(AuditEventTypes eventType) throws JsonProcessingException {
-        AuditEvent auditEvent = new AuditEvent((int) Instant.now().getEpochSecond(), eventType);
+    private String generateMessageBody(AuditEventTypes eventType, AuditExtensionParams extensions)
+            throws JsonProcessingException {
+        AuditEvent auditEvent =
+                new AuditEvent((int) Instant.now().getEpochSecond(), eventType, extensions);
 
         ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I know it's not your change, but can this objectMapper be made static? Newing one up for every audit event is going to be a bit of a drain I think.

Reply:
Yep will do 👍